app/services/reject_application.rb

```diff
@@ -0,0 +1,19 @@
+class RejectApplication
+  Response = Struct.new(:successful?, :application_choice)
+
+  def initialize(application_choice:, rejection:)
+    @application_choice = application_choice
+    @rejection = rejection
+  end
+
+  def call
+    @application_choice.status = :rejected
```

Comment:
This can now use the state machine like this:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/c8efbd590c5f54050378b77779420c2cf3a7e748/app/services/confirm_enrolment.rb#L9

Reply:
Cool. Does the state machine have a `reject` function? Or should I add that here :)

Reply:
Nope, that's up to you! :)